applications/openshift/api-server/audit_log_forwarding_enabled/rule.yml

```diff
@@ -20,7 +20,7 @@ references:
   nerc-cip: CIP-003-8 R5.2,CIP-004-6 R2.2.2,CIP-004-6 R2.2.3,CIP-004-6 R3.3,CIP-007-3 R.1.3,CIP-007-3 R5,CIP-007-3 R5.1.1,CIP-007-3 R5.2,CIP-007-3 R5.3.1,CIP-007-3 R5.3.2,CIP-007-3 R5.3.3,CIP-007-3 R6.5
   nist: AC-2(12),AU-3(2),AU-5(1),AU-6,AU-6(1),AU-6(3),AU-9(2),SI-4(16),AU-4(1),AU-11,AU-7,AU-7(1),SI-4(20)
   pcidss: Req-2.2,Req-10.5.3,Req-10.5.4
-  srg: SRG-APP-000181-CTR-000485,SRG-APP-000358-CTR-000805,SRG-APP-000092-CTR-000165,SRG-APP-000111-CTR-000220
+  srg: SRG-APP-000092-CTR-000165,SRG-APP-000111-CTR-000220,SRG-APP-000358-CTR-000805
```

Comment:
wondering if `SRG-APP-000358-CTR-000805` is supposed to be here, It is matching with the draft but is not in the export SRG file

Reply:
hmm, maybe there were some other patches merged in the meantime that were interfering. Let me just rebase these on master so that we see the latest export.

Reply:
oh sorry only now did I see what you meant, it should be fixed now

Reply:
umm, bad day. No, you are right, 805 shouldn't be there, this is one of the controls where we were going back and forth in the draft. The control should be not applicable. I'll fix this.